app/components/v2/scheduled_train_component.html.erb

```diff
@@ -1,47 +1,90 @@
-<%= render V2::SectionComponent.new(style: :titled, title: "Schedule") do %>
-  <div class="grid grid-cols-5 gap-1 text-main-500 dark:text-main-400">
-    <div class="pb-1 pl-0.5 heading-5">Past</div>
-    <div class="pb-1 pl-0.5 heading-5">Previous</div>
-    <div class="pb-1 pl-0.5 heading-5">Current</div>
-    <div class="pb-1 pl-0.5 heading-5">Next</div>
-    <div class="pb-1 pl-0.5 heading-5">Future</div>
+<%= render V2::SectionComponent.new(style: :titled, title: "Schedule", subtitle: release_schedule(train)) do %>
+  <div class="grid grid-cols-5 bg-main-500/80 divide-x divide-main-200 text-center py-2 rounded-t-lg">
+      <div class="pl-0.5 text-xs uppercase tracking-wider font-medium text-white dark:text-main-400">Past</div>
+      <div class="pl-0.5 text-xs uppercase tracking-wider font-medium text-white dark:text-main-400">Previous</div>
+      <div class="pl-0.5 text-xs uppercase tracking-wider font-medium text-white dark:text-main-400">Current</div>
+      <div class="pl-0.5 text-xs uppercase tracking-wider font-medium text-white dark:text-main-400">Next</div>
+      <div class="pl-0.5 text-xs uppercase tracking-wider font-medium text-white dark:text-main-400">Future</div>
   </div>
 
-  <div class="grid grid-cols-5 bg-main-50 border border-main-200 divide-x divide-main-200 rounded-lg">
+  <div class="grid grid-cols-5 border border-main-200 divide-x divide-main-200 rounded-b-lg">
     <% if past_releases.empty? %>
-      <div class="h-20 sm:h-28 lg:h-36 p-1 col-span-2">
-        <div class="h-full flex flex-col justify-between">
-          <!-- Events -->
-          <div class="grow flex flex-col p-0.5 sm:p-1.5 overflow-hidden">
-            NO PAST RELEASES
-          </div>
-        </div>
-      </div>
+      <%= render partial: "scheduled_release/show", locals: {
+        title: nil,
+        header: "There are no past releases",
+        empty_icon: "v2/fast_forward.svg",
+        box_classes: nil,
+        status: inactive_status,
+        footer: nil,
+      } %>
+      <%= render partial: "scheduled_release/empty", locals: { header: "No past releases"} %>
     <% else %>
       <% past_releases.each do |scheduled_release| %>
-        <%= render partial: "scheduled_release/show", locals: { header: past_text(scheduled_release), title: past_title(scheduled_release), status: status(scheduled_release), footer: time_format(scheduled_release.scheduled_at, only_day: true) } %>
+        <%= render partial: "scheduled_release/show", locals: {
```

Comment:
we should show a tooltip with the reason the release was skipped since the user might be confused as to why the scheduled release was skipped